src/components/generic/editable-field-components/MetaformNumberFieldComponent.tsx

```diff
@@ -0,0 +1,120 @@
+import { FormControl, Input, InputLabel, WithStyles, } from "@material-ui/core";
+import React from "react";
+import { Metaform, MetaformField } from "../../../generated/client";
+import strings from "../../../localization/strings";
+import styles from "../../../styles/form-edit-screen";
+
+/**
+ * Component props
+ */
+interface Props extends WithStyles<typeof styles> {
+  field: MetaformField;
+  fieldId: string;
+  fieldLabelId: string;
+  metaform: Metaform;
+  onValueUpdate: (key: string, number: number) => void;
+}
+
+/**
+ * Component state
+ */
+interface State {
+}
+
+/**
+ * Component for Metaform number field
+ */
+export class MetaformNumberFieldComponent extends React.Component<Props, State> {
+
+  /**
+   * Constructor
+   * 
+   * @param props component props
+   */
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+    };
+  }
+
+  /**
+   * Component render method
+   */
+  public render = () => {
+    const { classes, field, fieldId } = this.props;
+
+    const inputProps = {
+      min: field.min,
+      max: field.max,
+      step: 1
```

Comment:
Why hard coded step?

Reply:
For some reason the test Metaform json doesn't have step as property of field object. I now added the step as an optional value (with 1 been the other option so still hard coded) but this will be fixed on next task when we refine the UI of editor fields.